src/goog.js

```diff
@@ -28,7 +28,7 @@ define(['async', 'propertyParser'], function (async, propertyParser) {
 
                 settings.callback = onLoad;
 
-                req(['async!'+ (document.location.protocol === 'https:'? 'https' : 'http') +'://www.google.com/jsapi'], function(){
+                req([(document.location.protocol === 'https:'? 'https' : 'http') +'://www.google.com/jsapi'], function(){
```

Comment:
Change `www.google.com/jsapi` to `https://www.gstatic.com/charts/loader.js` (since that is where google is redirecting to anyway).

I don't believe the API actually changed, but the webserver hosting it did (the old one supported JSONP which is what ASYNC was using to load it...)

Reply:
I had also changed the URL, but the problem had not been fixed till I changed the async. Do you know anything about it?